scripts/jenkins/qa_devstack.sh

```diff
@@ -239,7 +239,7 @@ if [ -z "${DISABLE_TEMPESTRUN}" ]; then
 pip install junitxml
     sudo -u stack -i <<EOF
 cd /opt/stack/tempest
-tempest run --regex '(?!.*\[.*\bslow\b.*\])(^tempest\.(api|scenario))' --concurrency=2 --subunit > tempest.subunit
+tempest run --smoke --subunit | tee tempest.subunit | subunit-trace -f -n
```

Comment:
Why did you remove the ```--concurrency=2``` param?

Reply:
I added that parameter to limit the flakiness related to the concurrency when I enabled the regex with the upstream tempest tests, that increased considerably the number of tests.  
If you see the [trend on test failed per job ](https://ci.opensuse.org/view/OpenStack/job/openstack-devstack-ipv4/test/?width=800&height=600&failureOnly=true) there is still a lot of flakiness. Serializing avoid some flakiness but increase the time of each job, and currently, these jobs are taking more than 2 hours causing issues with other more important jobs as Colleen rightly pointed out.
Let's try to see the trend without that parameter now and change it if necessary later.

Reply:
@aojea that info would be useful in the commit message :) But to late - @JanZerebecki merged it already...

Reply:
you can see the commit history in the file and it explains the concurrency variable, there were more people asking for it when I introduced it

Reply:
sure. but there is no answer why you removed it now.

Reply:
I think it is ` switch back to the previous state where we only ran the smoke tempest tests.`, the previous state didn't have that parameter.